spec/python/run_ext2.py

```diff
@@ -13,19 +19,11 @@ def calc_dir(dir):
             sum += calc_dir(entry.inode.as_dir)
         elif entry.file_type == Ext2.DirEntry.FileType.reg_file:
             sum += entry.inode.size
-
     return sum
 
-t1 = timer()
-
-r = Ext2.from_file("%s/ext2.dat" % os.environ['DATA_DIR'])
-
-t2 = timer()
-
-sum = calc_dir(r.root_dir)
+calc_dir(r.root_dir)
 
 t3 = timer()
 
-print("sum = %d" % calc_dir(r.root_dir))
```

Comment:
The point of writing out the result of the job is to verify that it is doing the same kind of work that all other implementations do. Without it, you can't be sure.

Reply:
Benchmarks should (IMHO that is) measure speed, not correctness. We have an entire test suite for that. Construct benchmark suite does not check for correctness, if that matters.
https://github.com/construct/construct/blob/master/tests/test_benchmarks.py
https://github.com/construct/construct/blob/master/tests/test_all.py

Reply:
One of the points of having this benchmark is comparing languages implementations between each other. For that, we'll need a simple, standard, straightforward way to ensure that both C++, Java, Python, Ruby, etc, are doing the same close-to-real-world job.

And that's exactly what people ask very often: "How much would peformance improvement I'll get if I'll switch over to C++ or Java from my Ruby / Python code?"

Of course, Construct would not care about organizing benchmarks that way, as it only runs microbenchmarks — to compare different versions of modules to each other.

Reply:
Ah I get it now. I will update the PR to measure both parsing and processing speed. But I dont feel like it needs to do correctness checks. Thats test suite job.

Reply:
Then we'll need separate test suite for these benchmarks. Current approach is probably simplistic, yet if you want to have a test suite — sure, go ahead :)